.github/workflows/copy_to_gitbook_repo.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Copy File to Another Repo
-        uses: dmnemec/copy_file_to_another_repo_action@main
+        uses: nufi-official/copy_file_to_another_repo_action@main
```

Comment:
Fork to allow rsync_option, and delete excessive files on gitbook repo that can occur when f.e. renaming, deleting files 
https://github.com/nufi-official/copy_file_to_another_repo_action